checks/zookeeper_test.go

```diff
@@ -1,7 +1,8 @@
-package checks
+package checks_test
```

Comment:
It a common practice (for example -- https://github.com/golang/go/blob/master/src/strings/reader_test.go), But I postpone this one commit, and we can discuss it later (It not essential ;))